source/index.txt

```diff
@@ -120,6 +122,21 @@ Upgrade {+driver-short+} Versions
 Learn what changes you might need to make to your application to upgrade driver versions
 in the :ref:`pymongo-upgrade` section.
 
+Migrate from Motor to {+driver-async+}
+-------------------------------------
+
+In September 2024, MongoDB released the {+driver-async+} driver as a replacement
```

Comment:
The Async PyMongo is released yes, but it's only in beta (ie experimental) and not ready for production use. We should make this loud and clear.